.github/workflows/slash-command-dispatch.yaml

```diff
@@ -24,10 +30,19 @@ jobs:
     needs: [pullRequestStatus, checkAutomatedPR]
     if: ${{ needs.pullRequestStatus.outputs.isPullRequestMerged == 'False' && needs.checkAutomatedPR.outputs.isAutomatedPR == 'False' && !github.event.pull_request.head.repo.fork}}
     steps:
+      - name: Generate a token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
```

Comment:
looks like this is using third party action. Is there anything official action from github available for token generation?